src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -0,0 +1,307 @@
+﻿using System;
```

Comment:
Looks like some of the newer stuff got mixed into these changes--can you rebase it away, please?

Reply:
Oh! Didn't catch that. I will look into it. Thanks!